cmd/gin/app.go

```diff
@@ -2,24 +2,22 @@ package main
 
 import (
 	"flag"
-	"fmt"
 	"log"
 	"os"
 	"os/signal"
 
 	"github.com/Contrast-Security-OSS/go-test-bench/pkg/servegin"
 )
 
-// DefaultPort is the port that the API runs on if no command line argument is specified
-const DefaultPort = 8080
+// DefaultPort is the port that the API runs on if not overridden with '-addr' flag
+const DefaultPort = "localhost:8080"
```

Comment:
Can we rename this to `DefaultAddr` and update the doc comment?

Reply:
Updated, thanks!